### PR TITLE
add my exercise 14 files

### PR DESCRIPTION
**Pull request by bergado-shamma**

### CODE REVIEW

File: Bergado_Shamma/script/countries.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
+function searchCountry() {
+  let countryName = document.getElementById("country_input").value.trim();
```
Comment (Morcoso-daisy) [inline Bergado_Shamma/script/countries.js:2]: Try changing all the let to const. These codes are assigned once and are never reassigned. Using const ensures that the values are supposed to remain constant.

File: Bergado_Shamma/script/countries.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+              }</p>
+              <p><strong>Region:</strong> ${
+                country.region ? country.region : "N/A"
+              }</p>
+              <p><strong>Capital:</strong> ${
+                country.capital ? country.capital[0] : "N/A"
+              }</p>
+              <p><strong>Currency:</strong> ${
+                country.currencies
+                  ? Object.values(country.currencies).join(", ")
```
Comment (Reyes-PaulAngelo) [inline Bergado_Shamma/script/countries.js:42]: Observe proper structuring of elements

Follow this guidelines:
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g29e6ce31160_0_5

File: Bergado_Shamma/script/countries.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    .then(function (regionData) {
+      let region = regionData[0].region;
+      let sameRegionCountriesList = regionData
+        .map(function (c) {
+          return `
+                <div class="country-card">
+                  <img src="${c.flags.svg}" alt="Flag of ${c.name.common}"
+                   width="50">
+                  <p>${c.name.common}</p>
+                </div>
```
Comment (Reyes-PaulAngelo) [inline Bergado_Shamma/script/countries.js:81]: Remove unnecessary breaking of lines, Place line break in proper place

Follow this guideline
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g29e6ce31160_0_27

File: Bergado_Shamma/script/countries.js [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+              `;
+        })
+        .join("");
+      document.getElementById("same_region_countries").innerHTML = `
+              <h2>Countries in the Same Region (${region})</h2>
+              <div class="country-list">${sameRegionCountriesList}</div>
```
Comment (Reyes-PaulAngelo) [inline Bergado_Shamma/script/countries.js:87]: Remove unnecessary breaking of lines, Place line break in proper place

Follow this guideline
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g29e6ce31160_0_27

**Review by Reyes-PaulAngelo — COMMENTED** [review]
follow the guidelines

**Review by Reyes-PaulAngelo — APPROVED** [review]

**Pull request merged**